docs/README.md

```diff
@@ -22,6 +22,18 @@ To run ImageLab from the command line:
   
  Note that on Windows platforms, the ":" character in the classpath must be changed to the ";" character.)
 
+ ### Run with BlueJ
+[Install BlueJ for your operating system](https://www.bluej.org/)
+* Clone the ImageLab repository to your local machine.
+* Run BlueJ
+* Select Project in the BlueJ menu > Open Project
```

Comment:
I think this needs a little more common notation or perhaps more words. (I'm also not sure if it appears differently on different operating systems.)

Showing a menu's submenu is often done like this:
Project > Open Project...

But it may be more clear to just say it in words, like "Select _Open Project..._ from the _Project_ menu"